dapr/proto/runtime/v1/dapr.proto

```diff
@@ -595,4 +605,35 @@ message UnlockResponse {
   }
 
   Status status = 1;
-}
\ No newline at end of file
+}
+
+message WorkflowReference {
+  string InstanceID = 1;
+}
+
+message GetWorkflowRequest {
+  WorkflowReference WorkflowInfo  = 1;
+  string WorkflowType = 2;
+  string InstanceID = 3;
+  string WorkflowComponent = 4; 
+}
+
+message GetWorkflowResponse {
+  WorkflowReference WorkflowInfo  = 1;
+  google.protobuf.Timestamp StartTime = 2;
+  string TaskQueue = 3;
```

Comment:
Will "GetWorkflowResponse" be returned when calling "GET" on any type of workflow? I.E., not just temporal but eventually all back ends? If so, we will need to change it slightly since "TaskQueue" seems to be specific only to temporal and not to all other backends.